### ISSUE TITLE
what is y_train in tensorflow

### ISSUE THREAD

**Issue by StudentUsingGithup** [tensorflow/tensorflow#44055]
what is the meaning of y_train in this line of code?
`(x_train, y_train), (x_test, y_test) = tf.keras.datasets.cifar10.load_data()`

**Comment by sourabhsingh282** [comment]
y_train is the set of labels of all the data present in x_train
similarly, y_test  is the set of labels of the data present in x_test

**Comment by mihaimaruseac** [comment]
This is a question for StackOverflow as it is not related to a bug in the code / feature request.
